### ISSUE TITLE
Incorrect directory structure created when creating bio-gem?

### ISSUE THREAD

**Issue by wwood** [rjpbonnal/bioruby-gem#8]
As of version 0.3.0 of the gem (haven't checked newer versions of bio-gem), this seems amiss:

```
$ biogem orthomcl
...
Jeweler has prepared your gem in bioruby-bioruby-orthomcl
```

Full context:

```
$ biogem orthomcl
    create  .gitignore
    create  Rakefile
    create  Gemfile
    create  LICENSE.txt
    create  README.rdoc
    create  .document
    create  lib
    create  lib/bio-bio-orthomcl.rb
    create  test
    create  test/helper.rb
    create  test/test_bio-bio-orthomcl.rb
    create  lib/bio-bio-orthomcl.rb
Jeweler has prepared your gem in bioruby-bioruby-orthomcl
/home/ben/.rvm/rubies/ruby-1.8.7-p352/lib/ruby/1.8/fileutils.rb:121:in `chdir': No such file or directory - bioruby-orthomcl (Errno::ENOENT)
    from /home/ben/.rvm/rubies/ruby-1.8.7-p352/lib/ruby/1.8/fileutils.rb:121:in `cd'
    from /home/ben/.rvm/gems/ruby-1.8.7-p352/gems/bio-gem-0.3.0/bin/biogem:17
    from /home/ben/.rvm/gems/ruby-1.8.7-p352/bin/biogem:19:in `load'
    from /home/ben/.rvm/gems/ruby-1.8.7-p352/bin/biogem:19
```

A temporary fix that seems to work is to rename the bioruby-bioruby-orthomcl folder as bioruby-orthomcl and re-run the above command, then delete the newly created bioruby-bioruby-orthomcl (not a method that is very well tested, though).

I noticed similar issues using either of these variants

```
$ biogem bio-one
$ biogem bioruby-two
```

**Comment by rjpbonnal** [comment]
The bug is generated by this Jeweler's commit: https://github.com/technicalpickles/jeweler/commit/d0e05e69f351a88efaf2190c47616aa3c8f700b9
The Added support for absolut paths in project name rewrite project_name which I alias in my lib, so a prefix duplication generate a conflict in the name space directory.
Fixed version 0.3.1

**Comment by wwood** [comment]
Cool, thanks for the quick turnaround time.
